### ISSUE TITLE
Suggestion: Add wiki example(s) on how to recover snapshots from remote backups

### ISSUE THREAD

**Issue by rwarren** [jimsalterjrs/sanoid#752]
I think some quick examples of how to actually recover backups from a remote server would be useful to add to the wiki.

----

I'm in a case where my main server has snapshots going back 2 months, and my backup server has snapshots going back a full year.  What I want to do is recover a snapshot from 3 months ago from the backup server to the primary server (since the backup does not have this anymore).

I'd like to use syncoid to do this (due to its various conveniences)... but syncoid does not deal with copying single snapshots.  I am now trying to figure how to do this slightly odd recovery with basic `zfs send` usage, and was surprised to see that there is nothing in the sanoid/syncoid on how to actually recover the backups that it so excellently sets up!

(if curious why I'm rolling back a whole 3 months, it is because this is a qcow2-based dev server that has been off for > 2 months after being broken, so all local snapshots are the broken state... and I want to restore it to > 2 months ago)

**Comment by gnordli** [comment]
Did you get this sorted?   
If you have the space on the backup server, I would create a copy of that dataset, just to make sure as mistakes happen.

zfs send just the snapshot you from the backup server to the server. 

Then do a syncoid back from the main server to the backup server.   It should roll back to the snapshot you sent over.  

If you wanted to keep more snapshots, then you can use the zfs send -I to send all incremental snapshots between two snapshots from the backup server to the main server.

**Comment by rwarren** [comment]
> Did you get this sorted?
Yep. Sorted.  I used mix of `zfs send -I` (which needs an original `zfs send` to start from) as well as some base copy of data out of the `.zfs/snapshot` dir.

**Comment by rsysadmin-com** [comment]
You can also run:

`zfs set mountpoint=/foobar POOL/dataset
`
and access the files from /foobar

My 2 cents...
